ddtrace/ext/ci/fixtures/bitbucket.json

```diff
@@ -0,0 +1,374 @@
+[
```

Comment:
All of this `ci` folder belongs in `contrib/testing`. Let's move it there please. The `ext` package is not really an idiomatic approach but since we're using it now it's too late to remove it. It's fine to keep `ci.go` here with the tag names, but everything else needs to go into the `contrib/testing` package

Reply:
What about other testing frameworks that would like to use the CI integration?

Reply:
How about we start with `contrib/testing/internal/ci`? And then if there is a request from users to make it public, we can make it public.